atomic_reactor/plugins/pre_change_from_in_df.py

```diff
@@ -82,33 +78,14 @@ def run(self):
             # with the Dockerfile or some other code bug
             raise ParentImageMissing("Lost parent image(s) from Dockerfile: {}".format(missing_set))
```

Comment:
I know you didn't touch it, but can you change 

"from Dockerfile" to "from dockerfile_images"

Reply:
updated.